modules/microshift-kubeconfig-local-access.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_networking/microshift-kubeconfig.adoc
+
+:_content-type: CONCEPT
+[id="local-access_{context}"]
+= Local access kubeconfig file
+
+The local access `kubeconfig` file is generates in the `/var/lib/microshift/resources/kubeadmin/kubeconfig` directory. This configuration file allows access to the local API server using `localhost` and the internal CA of `localhost`. The CA of the local access `kubeconfig` file cannot be used outside of the {product-title} host.
```

Comment:
```suggestion
The local access `kubeconfig` file is generated in the `/var/lib/microshift/resources/kubeadmin` directory. This configuration file allows access to the local API server using `localhost` and the internal certificate authority (CA) of `localhost`. The CA of the local access `kubeconfig` file cannot be used outside of the {product-title} host.
```

@pacevedom is that right, the local kubeconfig file can't be copied to another host? If so, we should be more explicit about that in the statement here for someone who does not know what a "CA" is.

Reply:
Yes, this file is only valid for localhost access now, it does not work if you copy it to a different host.
We can include that each access type (local, remote) is served a different certificate, each signed by a different CA and that is what the different kubeconfigs are for.

I would also remove the `local` in `allows access to the local API server`.